deepsampler-junit/src/main/java/de/ppi/deepsampler/junit/JUnitPersistenceUtils.java

```diff
@@ -90,22 +95,110 @@ private static JsonSourceManager createSourceManagerWithJsonSerializerExtensions
         final JsonSourceManager.Builder persistentSampleManagerBuilder = JsonSourceManager.builder();
 
         applyJsonSerializersFromTestCaseAndTestFixture(testMethod, persistentSampleManagerBuilder);
+        Optional<SampleRootPath> rootPath = loadSampleRootPathFromTestOrSampleFixture(testMethod);
+
+        switch (loadSamples.source()) {
+            case FILE_SYSTEM:
+                Path file = createPathForFilesystem(rootPath, loadSamples.packagePath(), loadSamples.fileName(), testMethod);
+                return persistentSampleManagerBuilder.buildWithFile(file);
```

Comment:
Maybe a check if somebody uses a classpath instead of a filepath to provide a better errormessage